grpc/src/tx.rs

```diff
@@ -130,15 +128,14 @@ where
     /// # async fn docs() {
     /// use celestia_grpc::{TxClient, TxConfig};
     /// use celestia_proto::cosmos::bank::v1beta1::MsgSend;
-    /// use celestia_types::state::{AccAddress, Coin};
+    /// use celestia_types::state::{Address, AccAddress, Coin};
     /// use tendermint::crypto::default::ecdsa_secp256k1::SigningKey;
     ///
     /// let signing_key = SigningKey::random(&mut rand_core::OsRng);
-    /// let public_key = *signing_key.verifying_key();
-    /// let address = AccAddress::new(public_key.into()).into();
+    /// let address : Address = AccAddress::new((*signing_key.verifying_key()).into()).into();
```

Comment:
This doesn't look convinient. I suggest these explicit methods:

* `AccAddress::from_verifying_key`
* `ValAddress::from_verifying_key`
* `ConsAddress::from_verifying_key`
* `Address::from_account_verifying_key`
* `Address::from_validator_verifying_key`
* `Address::from_consensus_verifying_key`


Also, should we reexport `VerifyingKey` from `k256`?